pyemma/coordinates/tests/test_featurizer.py

```diff
@@ -245,7 +245,7 @@ def test_dihedrials_cossin(self):
         self.assertEqual(len(desc), self.feat.dimension())
 
     def test_backbone_dihedrals(self):
-        self.feat = MDFeaturizer(topfile=self.asn_leu_pdbfile)
+        self.feat = MDFeaturizer(topology=self.asn_leu_pdbfile)
```

Comment:
example for API breakage, because if arg is passed by the old name, this will fail